nilearn/maskers/nifti_labels_masker.py

```diff
@@ -252,6 +255,34 @@ def __init__(
 
         self.cmap = kwargs.get("cmap", "CMRmap_r")
 
+    def _sanitize_labels(self, labels):
+        """Clean up labels.
+
+        TODO this should ultimately be removed in favor of:
```

Comment:
TODO's should be written as an issue rather then left in the code ;-)

Reply:
agreed.

there is a bit of wrap up to do on this one!